src/applications/mhv/landing-page/components/LandingPage.jsx

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import CardLayout from './CardLayout';
+import NoHealthAlert from './NoHealthAlert';
 import HeaderLayout from './HeaderLayout';
 import HubLinks from './HubLinks';
 import NewsletterSignup from './NewsletterSignup';
+import { hasHealthData } from '../selectors';
 
 const LandingPage = ({ data = null }) => {
```

Comment:
You can clear this warning with prop-types...

```
import PropTypes from 'prop-types';
//...
LandingPage.propTypes = {
  data: PropTypes.object,
};
```

Reply:
Done